packages/cf-style-container/src/index.js

```diff
@@ -37,4 +39,24 @@ const applyTheme = (ComponentToWrap, theme) => {
   return ThemedComponent;
 };
 
-export { createComponent, applyTheme, ThemeProvider, color };
+const createComponentStyles = (styleFunctions, component) => {
```

Comment:
Done. I've documented all `cf-style-container` functions and added a test for `createComponentStyles`.